src/reverse/Converter.h

```diff
@@ -172,6 +172,18 @@ struct ClassConverter : LuaRED<ClassReference, "ClassReference">
         {
             result.value = aObject.as<ClassReference*>()->GetHandle();
         }
+        else if (aObject == sol::nil)
+        {
+            RED4ext::CClass* pClass = static_cast<RED4ext::CClass*>(apRtti);
+            if (pClass && !pClass->flags.isAbstract)
+            {
+                result.value = pClass->AllocInstance();
```

Comment:
I am not sure this is the right way to do this considering we don't release memory.
Also if I remember correctly AllocInstance just creates the memory but doesn't call the constructor

Reply:
I copied the logic from `NewObject` for `pType->GetType() == RED4ext::ERTTIType::Class` case:
https://github.com/yamashi/CyberEngineTweaks/blob/master/src/scripting/Scripting.cpp#L284
https://github.com/yamashi/CyberEngineTweaks/blob/master/src/scripting/Scripting.cpp#L303
This should create an empty instance for out param, which will be added to the result set after execution `results.push_back(Scripting::ToLua(state, args[i]));` where `args[i]` is the result of conversion.

Reply:
Alright it's a bug that's more prevalent than I thought, will create another issue for this